api/models/trial.js

```diff
@@ -210,6 +211,13 @@ const Trial = BaseModel.extend({
 
       return discrepancies;
     },
+    is_registered() {
+      // Be aware that this depends on `source_id` to be accurate
+      // So if a trial has a `nct` source but a `pubmed` source_id, the bug is elsewhere
+      const registers = _.filter(this.sources, (source) => source.type === 'register')
+            .map((source) => source.id);
+      return registers.indexOf(this.attributes.source_id) > -1;
```

Comment:
I totally forgot about these cute little shorthands, kudos for reminding me about them! Pushing the new code in a minute.